### PR TITLE
Css changes

### PR DESCRIPTION
**Pull request by yuliakh15**

### CODE REVIEW

File: teamwork.css [hunk 1]
```diff
@@ -31,26 +33,35 @@ body {
...
 }
 
 .highlight {
-  border-bottom: 10px solid white; 
+  border-bottom: 10px solid white;
 }
 
-.icon1, .icon2 {
+.icon1,
+.icon2 {
```
Comment (lapc002) [inline teamwork.css:56]: here is the MR conflict . You need to resolve it first

File: teamwork.css [hunk 2]
```diff
@@ -83,19 +90,25 @@ p {
 
 .icons-p2 {
   display: flex;
-  justify-content: center;
+  align-items: center;
 }
 
 h4.semibold2 {
```
Comment (lapc002) [inline teamwork.css:91]: the same here

File: teamwork.css [hunk 3]
```diff
@@ -83,19 +90,25 @@ p {
...
 font-weight: 600;
-color: black ;
+>>>>>>> c6916082c5ad0b930fd06a33b4f6f7b6c4918081
 }
 
 .section3 {
   font-size: 16px;
   margin: 0px 0px 48px;
-  margin-bottom: 3rem! important;
+  margin-bottom: 3rem !important;
```
Comment (lapc002) [inline teamwork.css]: try to avoid important

File: teamwork.css [hunk 4]
```diff
@@ -134,48 +146,57 @@ p.general-info {
...
-  color: rgb(12 110 255 / 50%);;
+  color: rgb(12 110 255 / 50%);
 }
 
 /* responsive styles */
 
 @media (max-width: 1000px) {
-
-  .section1-part1, .section1-part2 {
+  .section1-part1,
```
Comment (lapc002) [inline teamwork.css:154]: conflict

File: index.html [hunk 5]
```diff
@@ -95,7 +95,7 @@ <h2 class="V-main-h2">Get started any way you&nbsp;want</h2>
     </div>
   </div>
 
-  <div class="container">
+  <div class="Content-2-main-container">
```
Comment (lapc002) [inline index.html]: please rename class to lowercase

File: teamwork.css [hunk 6]
```diff
@@ -1,10 +1,13 @@
-.container {
-  margin-left: 150px;
-  margin-right: 150px;
+.Content-2-main-container {
```
Comment (lapc002) [inline teamwork.css]: here as well

File: teamwork.css [hunk 7]
```diff
@@ -1,10 +1,13 @@
...
+ display: flex;
+ flex-direction: column;
+ align-items: center ;
+ justify-content: space-around;
 }
 
 body {
```
Comment (lapc002) [inline teamwork.css:8]: styles with body should be in one class . you can grab it from all files and move to style.css which is general file

Comment (yuliakh15) [reply teamwork.css:8]: doesn't let me commit this change

**Review by lapc002 — APPROVED** [review]

**Pull request merged**